### PR TITLE
CVSL-2255: Add endpoint to retrieve COM caseload.

### PR DESCRIPTION
**Pull request by justin-gardiner**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/Functions.kt [hunk 1]
```diff
@@ -3,3 +3,16 @@ package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service
 import org.springframework.core.ParameterizedTypeReference
 
 inline fun <reified T> typeReference() = object : ParameterizedTypeReference<T>() {}
+
```
Comment (justin-gardiner) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/Functions.kt:6]: All these were just copied from CVL. I can't see anyway of sharing stuff like this between projects at the moment. It looks like kotlin-lib is just for spring stuff etc.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/OffenderServiceTest.kt [hunk 2]
```diff
@@ -35,13 +41,51 @@ class OffenderServiceTest {
...
     OffenderService(
+      assessmentRepository,
       offenderRepository,
       prisonerSearchService,
       probationService,
       staffRepository,
       telemetryClient,
     )
 
+  @Test
```
Comment (justin-gardiner) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/OffenderServiceTest.kt:52]: I noticed this was missing a unit test

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/Functions.kt (hunk 1 above)
Comment (andrewrlee) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/Functions.kt:6]: not super worried about duplication for things like this..

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**